### ISSUE TITLE
Feature Request - Add methods Purchase Authorize, Purchase Capture and Cancel Purchase

### ISSUE THREAD

**Issue by glennsdavis** [recurly/recurly-client-dotnet#611]
_Warning_: Github issues are not an official support channel for Recurly. If you have an urgent request we suggest you contact us through an official channel: support@recurly.com or https://recurly.zendesk.com

I have a feature request to add the methods listed below to v3 of the SDK. These methods exist in Recurly API v2.

- Purchase Authorize
- Purchase Capture
- Cancel Purchase

**Comment by joannasese** [comment]
Hi @glennsdavis. Apologies for the late response on this. We've shared your feature request with the appropriate parties, since it's currently outside of the scope of the client libraries. I'll close this request for now.

**Comment by glennsdavis** [comment]
Hi Joanna,
I requested that 3 methods from the v2 API (Purchase Authorize, Purchase Capture, Cancel Purchase) be included in the v3 API. What does it mean "it's currently outside of the scope of the client libraries"? Are these methods depreciated? It there another reason the methods were excluded from the v3 API? 
Thanks.

    On Friday, February 19, 2021, 01:58:40 PM PST, Joanna Sese <notifications@github.com> wrote:  
 
 


Hi @glennsdavis. Apologies for the late response on this. We've shared your feature request with the appropriate parties, since it's currently outside of the scope of the client libraries. I'll close this request for now.
